### ISSUE TITLE
Refresh button triggers Open command

### ISSUE THREAD

**Issue by jacobweber** [fabiospampinato/vscode-projects-plus#24]
Using vscode 1.23.0 and Projects+ 1.17.1:

When I click the projects icon on the sidebar, and then click the Refresh icon, it doesn't appear to refresh my projects. Instead it triggers the "Project: Open" command (it prompts me to "Select a project").

The only custom settings I have are:

```
	"projects.refreshRoots": [...],
	"projects.configPath": "...",
```

**Comment by jacobweber** [comment]
Gotcha. Just IMO it's a little confusing to make the refresh button do that.

But as far as refreshing projects, it seems to be finding projects at a lower depth than my refreshDepth is set to (or at least, not removing them from the projects list when I lower the refreshDepth). I can open a separate issue if I can reproduce it.

Is there a way to make it just list all folders at a certain depth as projects, even if they're not git / SVN / etc. projects?

**Comment by fabiospampinato** [comment]
@jacobweber yeah I guess it might be a bit confusing, but the fact that it asks you to open a project is also a way to tell you that it has finished refreshing the list, other than, as I mentioned, I think that's what one usual wants to do after refreshing the list anway.

> (or at least, not removing them from the projects list when I lower the refreshDepth)

The extension doesn't remove projects from your configuration, that's because you might insert some manually, which maybe are not found under your `refreshRoots` directories, and the extension can't distinguish between these 2 cases.

> Is there a way to make it just list all folders at a certain depth as projects, even if they're not git / SVN / etc. projects?

That's not possible.
